src/annotator/highlighter.js

```diff
@@ -331,6 +331,16 @@ export function setHighlightsFocused(highlights, focused) {
     // `<hypothesis-highlight>` element.
     if (h.svgHighlight) {
       h.svgHighlight.classList.toggle('is-focused', focused);
+
+      // Ensure that focused highlights are drawn above un-focused highlights
+      // on the same page.
+      //
+      // SVG elements are rendered in document order so to achieve this we need
+      // to move the element to be the last child of its parent.
+      if (focused) {
+        const parent = /** @type {SVGElement} */ (h.svgHighlight.parentNode);
+        parent.append(h.svgHighlight);
```

Comment:
There is not much difference here (https://developer.mozilla.org/en-US/docs/Web/API/Element/append), but I tend to use more `appendChild` because it feels safer.